src/main/java/org/janelia/saalfeldlab/paintera/PainteraDefaultHandlers.java

```diff
@@ -441,11 +447,32 @@ public static void grabFocusOnMouseOver(final Node node)
 
 	public void toggleInterpolation()
 	{
-		final Source<?> source = sourceInfo.currentSourceProperty().get();
-		if (source == null) { return; }
-		final ObjectProperty<Interpolation> ip = sourceInfo.getState(source).interpolationProperty();
-		ip.set(ip.get().equals(Interpolation.NLINEAR) ? Interpolation.NEARESTNEIGHBOR : Interpolation.NLINEAR);
-		baseView.orthogonalViews().requestRepaint();
+		if (globalInterpolationProperty.get() != null)
+		{
+			globalInterpolationProperty.set(globalInterpolationProperty.get().equals(Interpolation.NLINEAR) ? Interpolation.NEARESTNEIGHBOR : Interpolation.NLINEAR);
+			baseView.orthogonalViews().requestRepaint();
+		}
+	}
+
+	private InvalidationListener createSourcesInterpolationListener()
+	{
+		return obs ->
+		{
+			if (globalInterpolationProperty.get() == null && !sourceInfo.trackSources().isEmpty())
```

Comment:
This would fail if `sourceInfo.trackSources().iterator().next()` has `null` for interpolation. I do not think that any sources with that behavior but we should be aware of this potential source of error.